.github/workflows/release.yml

```diff
@@ -227,7 +227,7 @@ jobs:
           context: .
           file: ./ops/docker/Dockerfile.op-exporter
           push: true
-          tags: ethereumoptimism/op-exporter:${{ needs.canary-publish.outputs.op-exporter }}
+          tags: ethereumoptimism/op-exporter:${{ needs.release.outputs.op-exporter }}
```

Comment:
Looks good to me, can also add `,ethereumoptimism/op-exporter:latest` if we want to publish the latest tag as well. Other services do that

Reply:
Added!